index.html

```diff
@@ -2322,11 +2322,48 @@ <h2>Privacy Considerations</h2>
       <section>
         <h3>Unlinkability</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are digital signature
-schemes that provide uncorrelatable digital signatures.
+        <p>
+When a digitally-signed payload contains data that is seen by multiple
+verifiers, it becomes a point of correlation. An example of such data is a
+shopping loyalty card number. Correlatable data can be used for tracking
+purposes by verifiers, which can sometimes violate privacy expectations. The
+fact that some data can be used for tracking might not be immediately apparent.
+Examples of such correlatable data include, but are not limited to, a static
+digital signature or a cryptographic hash of an image.
+        </p>
+
+        <p>
+It is possible to create a digitally-signed payload that does not have any
+correlatable tracking data while also providing some level of assurance that the
+payload is trustworthy for a given interaction. This characteristic is called
+<dfn>unlinkability</dfn> which ensures that no correlatable data are used in a
+digitally-signed payload while still providing some level of trust, the
+sufficiency of which must be determined by each verifier.
+        </p>
+
+        <p>
+It is important to understand that not all use cases require unlinkability.
+There are use cases where linkability and correlation are required due to
+regulatory or safety reasons, such as correlating organizations and individuals
+that are shipping and storing hazardous materials. Unlinkability is useful
+when there is an expectation of privacy for a particular interaction.
+        </p>
+
+        <p>
+There are at least two mechanisms that can provide some level of unlinkability.
+The first method is to ensure that every data value used in the message is
+never repeated in a future message. The second is to ensure that any repeated
+data value(s) provide adequate herd privacy such that it becomes practically
```

Comment:
```suggestion
The first method is to ensure that no data value used in the message is
ever repeated in a future message. The second is to ensure that any repeated
data value provides adequate herd privacy such that it becomes practically
```

Reply:
Applied in e0d07318018d369ec568f12b49598198e2b96f34.